panda/src/gobj/samplerState.I

```diff
@@ -14,6 +14,7 @@
 /**
  * Creates a new SamplerState initialized to the default values.
  */
+
```

Comment:
Should not add newline between docstring and method (this may have been added by mistake)

Reply:
Yep, it was. Return removed. I did have modifications to that class in earlier implementations.